src/pages/style.css

```diff
@@ -15,6 +15,9 @@
 @import '~@hashicorp/design-system-tokens/dist/devdot/css/helpers/elevation.css';
 @import '~@hashicorp/design-system-tokens/dist/devdot/css/helpers/typography.css';
 
+/* Theme token files */
+@import 'styles/themes/dark.css';
```

Comment:
This token sheet will load for .io sites as well. I'm not too worried about it, but just noting.

Reply:
Also, these tokens will be available in production, but since the theme provider is handled with the feature flag, no danger of them being used.